doc/doxygen/src/riot-overview.svg

```diff
@@ -0,0 +1,327 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg version="1.1" width="1270" height="390"
+     xmlns="http://www.w3.org/2000/svg"
+     xmlns:cc="http://creativecommons.org/ns#"
+     xmlns:dc="http://purl.org/dc/elements/1.1/"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:svg="http://www.w3.org/2000/svg">
+  <title>RIOT Overview</title>
+  <metadata>
+    <rdf:RDF>
+      <cc:Work rdf:about="Provides a general overview over RIOT's structure">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title>RIOT Overview</dc:title>
+        <cc:license rdf:resource="http://creativecommons.org/licenses/by-sa/3.0/" />
+        <dc:creator>
+          <cc:Agent>
+            <dc:title>Cenk Gündogan</dc:title>
```

Comment:
@cgundogan I wasn't sure where the special characters go in your name. Please check.